tests/Feature/SharingTestScenariosAbstract.php

```diff
@@ -57,28 +57,28 @@
  */
 abstract class SharingTestScenariosAbstract extends SharingTestBase
 {
-	/** @var bool the previously configured visibility for public photos */
+	/** @var bool -- the previously configured visibility for public photos */
```

Comment:
No, I know that @qwerty287 asked you to make it consistent, because you used `:` as a separator and `--` in another place. I also had seen that, but it wasn't sufficiently important to me to ask for changing it.

But now you have changed it everywhere and now I have to ask you to revert it back, because it is not good. The PHPdoc comments don't use any separator at all. This way all our documents would start with an annoying `--` in the generated documentation. This is _not_ JsDoc where such a separator is actually required.

We should stick to the docs. See here: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/var.html#var

This is not arbitrary. Please revert it everywhere.

Reply:
Done.

Reply:
See [765957d](https://github.com/LycheeOrg/Lychee/pull/1469/commits/765957d951b036678753a021369c8b40bbe65673)